new/LIT7359MGBadlFight.xml

```diff
@@ -0,0 +1,107 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7359MGBadlFight" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of St George: St George fights for King Zarʿa Yāʿqob in the war against Badlāy who had destroyed a church dedicated to the saint</title>
```

Comment:
Zarʾa (for searchability reasons)?

Reply:
(I've updated it, but this is only relevant for oxygen/file-internal searchability, not on the app)

Reply:
How so? At least when searching in fidal, the app does not automatically neutralise homophones, I believe